### ISSUE TITLE
About baseline implementation and results in the paper

### ISSUE THREAD

**Issue by ridiculouz** [wzhouad/ATLOP#14]
Hello,
it's really nice work! While I have some question on the results provided in the paper: 1) what is the difference between BERT_base and BERT-E_base? 2) in ablation study, what does "-Adaptive-Thresholding Loss" stand for? Does it mean that you use a TH class with random parameters?
Besides, will you release your implementation of BERT-E_base?
Thanks!

**Comment by wzhouad** [comment]
Hi,
(1) BERT-E is short for the "Enhanced BERT baseline". You can implement it by removing ``rs`` in ``model.py`` (line 98 and 99) and replacing the adaptive thresholding loss with binary cross entropy.
(2) "-Adaptive-thresholding loss" means using the adaptive thresholding strategy in inference (output all classes with larger logits than NA) while training the model with binary cross entropy. It demonstrates that adaptive thresholding only works in models trained with our loss function.

**Comment by ridiculouz** [comment]
Thx for your reply!
